cli/cloe_launch/commands/activate.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2023 Robert Bosch GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# pylint: disable=too-many-arguments
+
+"""
+Implementation for cloe-launch command activate.
+"""
+
+from typing import List
+
+import click
+
+from cloe_launch import Configuration
+from cloe_launch.exec import Engine
+from . import _options
+from ._options import cli_command
+
+@cli_command("activate")
+@_options.cache()
+@_options.conanfile()
+@_options.args()
```

Comment:
Yeah, you're probably right. I think I had it public initially but something moved me to make it protected. I think it's because cloe is also supposed to be used as a library, in which case we don't want to export these CLI things. Not sure, it's been too long.

Reply:
After looking at it again, I will leave as is, since I don't want users of the cloe-launch library to access this "module". It's internal only.